packages/salesforcedx-vscode-core/src/conflict/conflictDetectionService.ts

```diff
@@ -238,3 +243,49 @@ export class ConflictDetector {
     telemetryService.sendException('ConflictDetectionException', errorMsg);
   }
 }
+
+export async function diffOneFile(
```

Comment:
let's use the impl from single file diff. there's an additional handling on the xml file. https://github.com/forcedotcom/salesforcedx-vscode/pull/3240/files#diff-8f671695467c60e6280f468736fe5827a3698ce526e3a37d292ef3a149899b8eR24-R69